lib/ftw/connection.rb

```diff
@@ -349,7 +351,11 @@ def secure(options=nil)
     sslcontext = OpenSSL::SSL::SSLContext.new
     # If you use VERIFY_NONE, you are removing the trust feature of TLS. Don't do that.
     # Encryption without trust means you don't know who you are talking to.
-    sslcontext.verify_mode = OpenSSL::SSL::VERIFY_PEER
+    if settings[:ssl_verify]
```

Comment:
Assuming I am convinced of this feature, I don't think this is the right change.

I think this change should instead use the verify_callback feature already present. The verify_callback can return false to reject a certificate and return true to accept. For example, here is now FTW::Agent does it: https://github.com/jordansissel/ruby-ftw/blob/master/lib/ftw/agent.rb#L439-L450